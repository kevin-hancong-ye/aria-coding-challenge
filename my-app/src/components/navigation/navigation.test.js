import React from "react";
import "@testing-library/jest-dom/extend-expect";
import { cleanup, render, screen } from "@testing-library/react";

import Navigation from "./navigation";

afterEach(cleanup);

test("renders navigation component", () => {
	render(<Navigation />);
	const backgroundImageElement = screen.getByRole("navigation");
	expect(backgroundImageElement).toBeInTheDocument();
	expect(backgroundImageElement).toMatchSnapshot();
});
