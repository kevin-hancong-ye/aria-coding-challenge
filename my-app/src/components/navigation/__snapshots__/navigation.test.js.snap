// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders banner component 1`] = `
<div
  class="navigationContainer"
  role="navigation"
>
  <ul>
    <li>
      <a
        href="#article_section"
      >
        Article
      </a>
    </li>
    <li>
      <a
        href="#accordian_section"
      >
        Accordian
      </a>
    </li>
    <li>
      <a
        href="#alert_dialog_section"
      >
        Alert Dialog
      </a>
    </li>
    <li>
      <a
        href="#checkbox_section"
      >
        Check Box
      </a>
    </li>
    <li>
      <a
        href="#table_section"
      >
        Table
      </a>
    </li>
  </ul>
</div>
`;
